rusk/tests/common/state.rs

```diff
@@ -97,6 +97,7 @@ pub fn generator_procedure(
         block_gas_limit,
         generator_pubkey,
         missed_generators,
+        timeout: Duration::from_secs(10),
```

Comment:
Magic number much?

Reply:
This is for test, so yes, a magic number I've choose

Reply:
Ah yes, the magic

Reply:
Removed the magic with latest commit